### PR TITLE
Remove Orchard.Hosting and upgrade components

### PR DESCRIPTION
**Pull request by Jetski5822**
Orchard.Hosting is kinda useless these days.
Upgrade components - cuz I can?

Renamed IOrchardHost to IShellHost

### CODE REVIEW

**Comment by agriffard** [comment]
CI checks have failed.

**Comment by Jetski5822** [comment]
@agriffard fixed

**Comment by Jetski5822** [comment]
Sweet - thanks @agriffard !!

File: src/OrchardCore/Microsoft.AspNetCore.Modules/Extensions/ModularServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -91,18 +90,18 @@ public static IServiceCollection AddModuleServices(this IServiceCollection servi
...
             this IServiceCollection services)
         {
-            return services.AddHost(internalServices =>
-            {
-                internalServices.AddLogging();
-                internalServices.AddOptions();
-                internalServices.AddLocalization();
-                internalServices.AddHostCore();
-                internalServices.AddExtensionManagerHost("App_Data", "dependencies");
+            services.AddLogging();
```
Comment (sebastienros) [inline src/OrchardCore/Microsoft.AspNetCore.Modules/Extensions/ModularServiceCollectionExtensions.cs]: formatting?

Comment (Jetski5822) [inline src/OrchardCore/Microsoft.AspNetCore.Modules/Extensions/ModularServiceCollectionExtensions.cs]: nope ;)

**Pull request merged**